### ISSUE TITLE
[Frontend] - Remove page blocks on the Settings Details tab

### ISSUE THREAD

**Issue by AdityaBoddepalli** [Northeastern-Electric-Racing/FinishLine#2146]
### Description

Blocked by: #2145
**Important:** Branch off and merge into the feature/settings-redesign branch

The details tab on the settings page has components wrapped in page blocks. Remove the page blocks for all the details on that tab. It should look similar to the mocks below when you are done. 

### Acceptance Criteria

- [ ] The functionality of the settings page should remain the same.
- [ ] There shouldn't be any page blocks on the details tab.
- [ ] The tab includes user details like name, email, teams, role, slack id etc.

### Proposed Solution

N/A

### Mocks

![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/68555142/8f2fc467-1c4e-4d0e-9094-aeebeaabc13d)
